cmd/scaler/server.go

```diff
@@ -418,23 +442,36 @@ func (s *httpServer) handleScaleUp(
 			if err := group.Wait(); err != nil {
 				return err
 			}
-			s.logger.Infon("All snapshots created")
+			log.Infon("All snapshots created", logger.NewStringField("duration", time.Since(start).String()))
+		} else {
+			log.Infon("Skipping snapshots creation")
 		}
 
 		// Step 4: Load snapshots to destination nodes
+		log.Infon("Scale up to start loading snapshots",
+			logger.NewIntField("totalNodesAffected", int64(len(destinationNodeMovements))),
+		)
+
+		start := time.Now()
 		group, gCtx := errgroup.WithContext(ctx)
 		for nodeID, hashRanges := range destinationNodeMovements {
 			if len(hashRanges) == 0 {
 				continue
 			}
 			group.Go(func() error {
 				return s.retryViaPolicy(gCtx, retryPolicy, func() error {
-					if err := s.scaler.LoadSnapshots(gCtx, nodeID, hashRanges...); err != nil {
+					start := time.Now()
```

Comment:
```suggestion
					nodeStart := time.Now()
```
current one is shadowing the start time above, makes it confusing to read